src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,117 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "book")
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+    private String genre;
+    @Column(nullable = false)
+    private String author;
+    @Column(nullable = false)
```

Comment:
Use NotNull annotation instead of nullable=false, to verify the null value before operate in database

Reply:
Change NotNull annotation instead nullable